<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInputDialogExport</class>
 <widget class="QDialog" name="CInputDialogExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>491</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Codec:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCodec">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>UTF-8 (Unicode)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Windows-1252</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Trennzeichen:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSplitChar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Texttrenner:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTextSpliter">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>78</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>&quot;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>'</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>kein</string>
    </property>
   </item>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditExamble">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>481</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonSemi">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Semikolon</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonCom">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Komma</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonTab">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Tabulator</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonSpace">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Leerzeichen</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonOther">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>anderes Zeichen</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBoxCodec</tabstop>
  <tabstop>comboBoxTextSpliter</tabstop>
  <tabstop>radioButtonSemi</tabstop>
  <tabstop>radioButtonCom</tabstop>
  <tabstop>radioButtonTab</tabstop>
  <tabstop>radioButtonSpace</tabstop>
  <tabstop>radioButtonOther</tabstop>
  <tabstop>lineEditSplitChar</tabstop>
  <tabstop>lineEditExamble</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CInputDialogExport</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CInputDialogExport</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
